R/mod_tableGen_fct_filter_adae.R

```diff
@@ -0,0 +1,173 @@
+filter_adae <- function(recipe, datalist, ADSL) {
```

Comment:
A good example of a DSL like this in the wild can be found in the [CDISC Analysis Results Standard GitHub repo](https://github.com/cdisc-org/analysis-results-standard). [Here's an example](https://github.com/cdisc-org/analysis-results-standard/blob/b5d61bc29205805faf80ef3584dad7fb3fa472c7/workfiles/examples/PHUSE%20Connect%20Workshop/PHUSE%20Connect%20Examples.json#L89-L101) of their metadata syntax to define a common population filter:

```json
    {
      "id": "AnalysisSet_02_SAF",
      "condition": {
        "dataset": "ADSL",
        "variable": "SAFFL",
        "comparator": "EQ",
        "value": [
          "Y"
        ]
      },
      "order": 1,
      "label": "Safety Population"
    }
```

I think it will be good for us to reference CDISC ARS conventions as we go through the process of defining our metadata specification.